RELEASING.md

```diff
@@ -0,0 +1,172 @@
+# AdoptOpenJDK Release Guide
+
+Don't be scared off by this document! If you already understand the stuff in the glossary section and are only working on a HotSpot release, then skip to [Steps for every version](#steps-for-every-version) later on.
+
+## Release glossary and background information
+
+### OpenJDK "New" Release:
+
+- The refers to "new" major OpenJDK release Short or Long Term Release (jdk13, jdk14, jdk15, ...)
+- Oracle and contributors work on release in "head" openjdk stream: https://hg.openjdk.java.net/jdk/jdk
+- 3 months prior to the GA date, the `head` stream if branched into the new release stream for development rampdown e.g. https://hg.openjdk.java.net/jdk/jdk14
+- Regular builds are tagged every week or so in a format such as  `jdk-13+21`
+- Eventually after rampdown and final phase testing, all done in the open, the GA build is tagged and released, `jdk-13-ga` tag is tagged along side the actual release build tag.
+
+
+### OpenJDK Vulnerability Group
+
+- A private group of trusted people who take reports of vulnerabilities in the openjdk codebase and work to resolve them and get them into releases in a timely manner
+
+### Non-Oracle (usually RedHat) maintained OpenJDK Quarterly PSU Update
+
+- Maintainers work on public quarterly update fixes for the next update, in the updates head stream e.g. https://hg.openjdk.java.net/jdk-updates/jdk11u
+- Fixes for the subsequent update, are developed in the `dev` stream e.g. https://hg.openjdk.java.net/jdk-updates/jdk11u-dev/
+- Regular builds are tagged every week or so, e.g. `jdk-11.0.5+6`
+- Eventually after final testing it comes to GA day for the update, at this point any fixes from the Vulnerability Group are merged with the final GA build, and tagged, e.g. `jdk-11.0.5+10`, `jdk-11.0.5-ga`
+
+### Oracle Managed OpenJDK Quarterly PSU Update
+
+- For the most recent JDK release, updates are managed by Oracle, and there will only be two of them, e.g. `jdk-13.0.1` and `jdk-13.0.2` after the initial GA unless it is an LTS release
+- Oracle work on the quarterly updates internally and their these branches are not public until GA date along with the Vulnerability Group fixes.
+- On GA day, Oracle merges the internal branch and any Vulnerability Group fixes to produce the final GA build, and tagged, e.g. `jdk-13.0.1+9`, `jdk-13.0.1-ga`
+
+### Eclipse OpenJ9/OMR releases
+
+- The OpenJ9 releases are based on three codebases
+  - https://github.com/eclipse/openj9-omr (a platform abstraction layer which OpenJ9 builds use, based on https://github.com/eclipse/openj9-omr)
+  - https://github.com/eclipse/openj9 (The JVM code)
+  - The "extensions" repository for each release which contains a modified version of the OpenJDK codebase (OpenJDK updates are merged by the IBM team who owns the extensions repository, not the AdoptOpenJDK project)
```

Comment:
Useful to have an example link here, like https://github.com/ibmruntimes/openj9-openjdk-jdk which is the head stream (next version of openjdk)

Reply:
I'll add something like `see next glossary entry` where I have provided an example link into it. Was trying not to dilute that section by talking about extensions just yet. I did consider mergin both sections though